src/main/java/seedu/command/CompareCommand.java

```diff
@@ -26,13 +26,31 @@ public void execute(UserList users, Ui ui, User nowUser) throws WhereGotTimeExce
         if (nowUser == null) {
             throw new WhereGotTimeException("Sorry! You are not logged in to any account!");
         } else {
-            String[] parsedInputs = input.split(" /", 3);
-            String targetName = parsedInputs[0].substring(1);
-            String date = parsedInputs[1];
+            int tempInt = 1;
+            System.out.println("Please input the index number of the user you would like to compare with.");
+            System.out.println("____________________________________________________________");
+            for (User u : users.getUserList()) { 
+                System.out.println(tempInt + ". " + u.getName());
+                tempInt++;
+            }
+            System.out.println("____________________________________________________________");
+            Scanner scanner = new Scanner(System.in);
+            String tempString = scanner.nextLine().trim();
```

Comment:
Maybe can add exceptions here in case user enter a command not expected by the program.